banner_warn.php

```diff
@@ -142,15 +142,15 @@ private function addressExternal($address) {
         }
 
         private function spfFails($headers) {
-            $spfStatus = $this->first($headers->others[strtolower($this->received_spf_header)]);
```

Comment:
Wouldn't null coalescing have the same effect?

```
$spfStatus = $this->first($headers->others[strtolower($this->received_spf_header)] ?? null);
```

Reply:
:+1: 
old habits...
i'll redo my pr